### ISSUE TITLE
Question About Training Code

### ISSUE THREAD

**Issue by FeiSong123** [csslc/PiSA-SR#21]
In the paper's network diagram, "pix" and "sem" appear to be two separate branches generating two distinct outputs, but they are merged in the training code. Why is this the case?

**Comment by XDUcy** [comment]
From the code, the author's idea should be to train Pix Lora for only 4000 steps, and then change all the training steps to train SEM Lora, and keep the existing Pix Lora frozen. And before the training of the SEM layer begins, all the loss weights except L2 loss are 0

**Comment by FeiSong123** [comment]
Thank you. I got it.

**Comment by PhuTran1005** [comment]
Argument "max_train_steps" is set to default value is 100K. How can I update it to align with the setting of the paper?
